docs/tutorials/migrations/nvm.md

```diff
@@ -0,0 +1,122 @@
+---
+title: Node Version Manager (nvm)
+description: Using Flox to replace nvm
+---
+
+# Node Version Manager (nvm) migration guide
+
+Flox is an environment and package manager that allows you install software from an extensive catalog into individual [environments][environment_concept]{:target="\_blank"}, each of which usually corresponds to a software project. Flox gives you the opportunity to simplify the development workflow used by your team, consolidating various functions into a single command: `flox activate`.
+
+Just like [nvm](https://github.com/nvm-sh/nvm){:target="\_blank"}, Flox allows you to install and activate different versions of Node.js, allowing developers to switch versions as project requirements dictate.
+
+## Why you might want to use Flox instead of nvm
```

Comment:
I think you're missing something about reproducibility.

Reply:
I'm probably missing something, but I think nvm will give you the same reproducibility that Flox does, at least insofar as node versions are concerned. (e.g., The `.nvmrc` will ensure that you have the node version you want.) There are gaping holes in my knowledge on this subject, so there may be something big that I've simply glossed over!